.github/workflows/main.yml

```diff
@@ -1,6 +1,18 @@
-# Copyright (c) 2020-2024 Digital Bazaar, Inc.
+# SPDX-FileCopyrightText: 2020 - 2024 Digital Bazaar, Inc.
```

Comment:
- Why is this one file using the SPDX tag, but not others?
- Why exclude the "Copyright" word?  It's easier to read and search for that way.
- Why the year range spaces?  Their examples seem to all have no spaces, which is the style we've used forever.

Reply:
An accident. I'll fix it.

I've no idea about the whitespace. I don't like it either, personally, but would rather not fork it. Feel free to ask about it upstream: https://github.com/fsfe/reuse-tool/issues